.bumpversion.cfg

```diff
@@ -0,0 +1,20 @@
+[bumpversion]
+current_version = 2.1.0
+tag = False
+commit = False
+
+[bumpversion:file:pyproject.toml]
+search = 
+	name = "inmanta-nfv-test-api"
+	version = "{current_version}"
+replace = 
+	name = "inmanta-nfv-test-api"
+	version = "{new_version}"
+
+[bumpversion:file:src/nfv_test_api/__init__.py]
```

Comment:
Not sure these files require custom search and replace but it can do no harm either.

Reply:
How else would you do it?

Reply:
The default just searches for the version and replaces it with the version so you could just leave the sections empty.

Reply:
Oh, nice